packages/notification-service/package.json

```diff
@@ -43,6 +43,8 @@
     "@types/utf8": "^2.1.6",
     "@types/web3": "^1.0.18",
     "jest": "^24.8.0",
-    "ts-jest": "^24.0.0"
+    "ts-jest": "^24.0.0",
+    "ts-node": "^7.0.1",
+    "typescript": "^3.3.3"
```

Comment:
Yeah this should have been down here, please remove line 30 from the dependencies section